CHANGELOG.rst

```diff
@@ -2,6 +2,12 @@
 Changelog for package slam_karto
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.7.2 (2015-06-02)
```

Comment:
you don't need to update this file -- it is part of the release process (which is automated)